### DISCUSSION TITLE
cannot import name 'digit_version' from 'mmocr' (unknown location)

### DISCUSSION THREAD

**Discussion by HardikJain02** [open-mmlab/mmocr#1935] [General]
When I run this training command: "python mmocr/tools/train.py mmocr/configs/textdet/dbnet/dbnet_resnet18_fpnc_1200e_icdar2015.py" 

this error shows up:  cannot import name 'digit_version' from 'mmocr' (unknown location)

help!

**Comment by Ivandanilovich** [comment]
It should import digit_version from mmocr/__init__.py. For some reason system can not find path. 
Simple hack to overcome it is to add path directly. Before all mmocr imports: 
`sys.path.append(os.sep.join([<path to mmocr folder location>, 'mmocr']))`
Maybe It's not elegant way, but it works.
